src/client/components/modals/index.js

```diff
@@ -188,6 +200,11 @@ const UserInfoModal = ({ ethereumAddress, onSignatures }) => {
             signaturesRequired={modal.sigsRequired}
           />
         )}
+        {modal.type === 'savingProfile' && <AfterSave savingProfile />}
+        {modal.type === 'savedProfileSuccess' && (
+          <AfterSave savedProfileSuccess />
+        )}
+        {modal.type === 'savedProfileError' && <AfterSave savedProfileError />}
```

Comment:
Since this whole codebase deals with profiles, is it redundant to put `Profile` in each of these? Could we name them something simpler, like `saving`, `success`, and `error`?

Also, while I'm commenting on this file, here are a couple ideas for making this file more robust in general, which we can address at a later time:

* It would be really easy to introduce bugs via misspellings right now. Instead of using the same strings everywhere, let's make the names of these modals constants. That will also help explain where this logic is coming from, in the places where we open modals.

* This:

  ```js
  {cond1 && <Comp1 />}
  {cond2 && <Comp2 />}
  {cond3 && <Comp3 />}
  ```

  Seems less clear to me than using a function or an object to return the correct component.  

  ```js
  const modals = {
    [SAVING]: <AfterSave mode={SAVING} />,
    [SUCCESS]: <AfterSave mode={SUCCESS} />,
    [ERROR]: <AfterSave mode={ERROR} />,
  }
  ...
  {modals[modal.type]}
  ```